hr_attendance_modification_tracking/tests/test_hr_attendance.py

```diff
@@ -83,7 +83,9 @@ def test_attendance_edit(self):
         dto = datetime.now() + relativedelta(hours=1)
         att.write({"check_out": dto.strftime(DF)})
         self.assertEqual(
-            att.time_changed_manually, True, "Use case 5: Wrong value, data changed."
```

Comment:
I don't think changing this assert from True to False is something valid. This shows the behavour has been changed in the migration.